src/Renderer/ThreeExtended/GeoJSON2Features.js

```diff
@@ -26,21 +26,25 @@ function readCRS(json) {
     return 'EPSG:4326';
 }
 
+const coords = new Coordinates('EPSG:4978', 0, 0, 0);
 function readCoordinates(crsIn, crsOut, coordinates, extent) {
     // coordinates is a list of pair [[x1, y1], [x2, y2], ..., [xn, yn]]
-    const out = [];
+    const out = new Array(coordinates.length);
+    let i = 0;
     for (const pair of coordinates) {
         // TODO: 1 is a default z value, makes this configurable
-        const coords = new Coordinates(crsIn, pair[0], pair[1], 1);
         if (crsIn === crsOut) {
-            out.push(coords);
+            out[i] = new Coordinates(crsIn, pair[0], pair[1], 1);
         } else {
-            out.push(coords.as(crsOut));
+            coords.set(crsIn, pair[0], pair[1], 1);
+            coords._normal = undefined;
```

Comment:
This is a no from me, a variable prefixed with an underscore is a private value AFAIK, so we shouldn't access it like that. You should tweak `Coordinates.js` if you need to change this value.

Reply:
yes you right, now `Coordinates.set` undefines `_normal`